frontend/public/components/RBAC/edit-rule.jsx

```diff
@@ -53,7 +53,16 @@ const RadioButton = ({name, value, label, text, onChange, activeValue}) => <div>
 const HRMinor = () => <hr className="rbac-minor" />;
 const HRMajor = () => <hr className="rbac-major" />;
 
-const EditRule = connect(state => state.k8s.get('RESOURCES') || {}, {getResources: k8sActions.getResources}) (
+const stateToProps = state => {
+  const resourceMap = state.k8s.get('RESOURCES');
+  let props = {};
+  resourceMap.forEach(function(value, key){
+    props[key] = value;
+  });
+  return props;
```

Comment:
Should this be `resourceMap.toObject()` or `resourceMap.toJS()`?

Reply:
Ha, yeah. I was reading the docs for vanilla JS Map, which doesn't have those functions. That's much better. Probably want to use `toObject` because I don't want any child Maps to be converted to objects.